File: app/src/main/java/com/rendersoncs/reportform/itens/ReportResumeItems.java
package com.rendersoncs.reportform.itens;

public class ReportResumeItems {

    private String title, description, conformity, note, photo;

    public ReportResumeItems(String title, String description, String conformity, String note, String photo){
        this.title = title;
        this.description = description;
        this.conformity = conformity;
        this.note = note;
        this.photo = photo;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getConformity() {
        return conformity;
    }

    public void setConformity(String conformity) {
        this.conformity = conformity;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getNote() {
        return note;
    }

    public void setNote(String note) {
        this.note = note;
    }

    public String getPhoto() {
        return photo;
    }

    public void setPhoto(String photo) {
        this.photo = photo;
    }
}

File: app/src/main/java/com/rendersoncs/report/model/ReportResumeItems.kt
package com.rendersoncs.report.model

import com.rendersoncs.report.common.util.ChecklistPhotos

class ReportResumeItems(
        val key: String,
        var title: String,
        var description: String,
        var conformity: Int,
        var note: String,
        var photo: String
    ) {
        val photoPaths: List<String>
            get() = ChecklistPhotos.parse(photo)
    }
